configure-aar.md

```diff
@@ -0,0 +1,117 @@
+The Anbox Application Registry (AAR) uses a certificate-based authentication system that uses TLS server and client certificates to establish a trusted connection between the AAR and AMS.
+
+AAR and AMS must exchange certificates to set up a trust relation. The recommended way to do this is with Juju, but it is also possible to do it manually.
+
+## Register an instance using Juju (recommended)
+
+Use [Juju relations](https://jaas.ai/docs/relations) to register an instance with the AAR.
+
+To register an instance as client, use the following command:
+
+    juju add-relation aar:client ams:registry-client
+
+To register an instance as publisher, use the following command:
+
+    juju add-relation aar:publisher ams:registry-publisher
+
+> **Hint:** Run `amc config show` to check that the AAR configuration items were changed.
+
+### Register units deployed in another model
+
+For `ams` units deployed in another model, you can make use of [Juju cross model relations](https://juju.is/docs/cross-model-relations).
+
+```sh
+$ juju switch <model containing aar>
+$ juju offer aar:client
+my-controller/my-model.aar
```

Comment:
Is this an output? I know too little about Juju to understand what we're doing here. Any suggestions for how to split this up?

Reply:
Yes, this is the actual output and the name of the offer the command above generated